android/app/src/main/python/mmrelay/plugins/drop_plugin.py

```
@@ -0,0 +1,127 @@
+            self.store_node_data(
+                self.special_node,
+                {
+                    "location": (position["latitude"], position["longitude"]),
+                    "text": drop_message,
+                    "originator": packet["fromId"],
+                },
+            )
+            self.logger.debug(f"Dropped a message: {drop_message}")
+            return True
+
+    async def handle_room_message(self, room, event, full_message):
```

Ruff (0.12.2)

Unused method argument: `room`

(ARG002)